docs/content/config/advanced/user-patches.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Custom user changes & patches'
```

Comment:
Title Case: "Custom User Changes & Patches".

Without a markdown heading, this metadata will be used to reflect the pages title (shown in the browser title), we usually prefix this with a category for context as you might see around other docs. If you move this to the "Override the Default Configs" category, look at the sibling documents and just copy/paste that title prefix.

The document title/heading on the page itself will iirc, use the assigned navigation label, or if given an H1 heading in the markdown document prioritize that if you'd rather something different from the nav label.

Reply:
Done. I have appended `| Scripting` as metadata.